foca/models/config.py

```diff
@@ -528,15 +541,22 @@ class MongoConfig(FOCABaseConfig):
 
     Example:
         >>> MongoConfig(
-        ...     host="mongodb",
-        ...     port=27017,
+        ...     host="mongodb_host",
```

Comment:
Why this name changes? `mongodb` is what we will usually call our host in Docker. While it might make sense here (to be more explicit), it doesn't make too much sense to have to call it host elsewhere.